transform/variant/maf_transform.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" transform a maf file into vertexs[variant, allele]   """
+
+import argparse
+import logging
+import csv
+import gzip
+import sys
+
+from bmeg.models.vertex_models import Biosample, Callset
+from bmeg.models.edge_models import AlleleCall, CallsetFor
+from bmeg.models.emitter import Emitter
+
+import allele_harvester
+
+from concurrent.futures import ThreadPoolExecutor
+from concurrent.futures import as_completed
+
+
+def get_value(d, keys, default):
+    """ utility get value from list"""
+    if isinstance(keys, list):
+        for k in keys:
+            if k in d:
+                return d[k]
+    elif keys in d:
+        return d[keys]
+    return default
+
+
+def _multithreading(func, lines, max_workers=5):
+    """
+    Create a thread pool and create alleles
+    """
+    with ThreadPoolExecutor(max_workers=max_workers) as executor:
+        futures = (executor.submit(func, line) for line in lines)
+        for future in as_completed(futures):
+            yield future.result()
+
+
+def _read_maf(mafpath, gz):
+    """ generator for each line in maf """
+    if gz or 'gz' in mafpath:
+        inhandle = gzip.open(mafpath, mode='rt')
+    else:
+        inhandle = open(mafpath)
+    reader = csv.DictReader(inhandle, delimiter="\t")
+    for line in reader:
+        yield line
+    inhandle.close()
+
+
+class MafConverter():
```

Comment:
The parentheses aren't needed, but you don't really need a class either. Just make it a function and merge it with the existing `convert()` please.

Reply:
I hear you re. the class.   I was attempting to re-work the [original](https://github.com/bmeg/bmeg-etl/blob/master/transform/variant/ga4gh-variant.py)  more than write something new.